spec/features/staff/users_can_view_reports_spec.rb

```diff
@@ -156,6 +161,9 @@
       expect(page.response_headers["Content-Type"]).to include("text/csv")
       header = page.response_headers["Content-Disposition"]
       expect(header).to match(/#{ERB::Util.url_encode("#{report.organisation.beis_organisation_reference}-report.csv")}\z/)
+
+      contents = CSV.parse(page.body, headers: true).map { |column| column["Title"] }
+      expect(contents).not_to be_empty
```

Comment:
If we're testing it, should we test a bit more than just "not empty"? How do we know it actually contains what it should? 😬

Reply:
Fair point! I've used a more exacting assertion now...